static_src/test/server/fixtures.js

```diff
@@ -165,9 +183,30 @@ var organizations = organizationGuids.map(function(guid) {
 
 module.exports.organizations  = organizations;
 
+var domains = domainGuids.map(function(guid, i) {
+  var domainType = (i % 2 === 0) ? 'private_domains' : 'shared_domains';
```

Comment:
you have this logic here and on line 207. maybe make a helper function like `getDomainType(i)`